.github/workflows/cypress.yml

```diff
@@ -27,10 +27,9 @@ jobs:
         ports:
           - 6379:6379
       ckan-solr:
-        image: solr:6.5
+        image: ckan/ckan-solr:master
```

Comment:
Should the docker-compose SolR Dockerfile also use `ckan:ckan-solr:master`? 
The [first two lines](https://github.com/ckan/ckan/blob/master/contrib/docker/solr/Dockerfile#L1-L2) would become (with an updated `LABEL`): 
```
FROM ckan/ckan-solr:master
LABEL maintainer="Open Knowledge"
```

Reply:
@florianm no, I think we should completely remove the solr Dockerfile in this repo and call the `ckan/ckan-solr` image directly from the docker-compose file:

```diff
diff --git a/contrib/docker/docker-compose.yml b/contrib/docker/docker-compose.yml
index 3fe96ddee..7687f9b46 100644
--- a/contrib/docker/docker-compose.yml
+++ b/contrib/docker/docker-compose.yml
@@ -65,9 +65,7 @@ services:
 
   solr:
     container_name: solr
-    build:
-      context: ../../
-      dockerfile: contrib/docker/solr/Dockerfile
+    image: ckan/ckan-solr:2.10
     volumes:
       - solr_data:/opt/solr/server/solr/ckan/data
     restart: unless-stopped
```

Maybe the volume path needs updating, I'm not sure.

Want to submit a PR with these changes?

Reply:
Ah, calling `ckan/ckan-solr` from the compose file makes a lot of sense.

I'm just working through the docker-compose approach and will likely have some fresh ideas by Feb next year (xmas break is looming). It appears that the docker-compose approach would benefit from the patch release and all the SolR changes from this PR.

I also feel that mounted folders would provide a smoother workflow for plugin development after all, which means re-working the docker-compose approach.

All of this means there's more work to do before I can submit a comprehensive PR for the docker-compose install, and I'll be slow about it too (Feb '22).

Did you want me to submit a quick PR for just the changes above for now? (Not sure about the volume path, strapped for time to figure out the correct path)